swarm/src/test/resources/docker/swarm.tests.bom

```diff
@@ -396,6 +396,7 @@ brooklyn.catalog:
         swarm.port: 4000
         docker.recovery.stabilizationDelay: 10s
         docker.recovery.failOnRecurringFailuresInThisDuration: 5m
+        swarm.sharedsecuritygroup.create: true
```

Comment:
@Graeme-Miller there should be tests with both _true_ and _false_ settings for this configuration key, on the appropriate clouds. But, to verify that the default setting work, we should *NOT* be setting it here, for the obvious reason that if the default was ever to change, we would not be "test[ing] what a user will get by default" any more.